src/helpers/utils.ts

```diff
@@ -95,6 +95,20 @@ export function filterSkins(skins, spaces, q) {
     .sort((a, b) => b.spaces.length - a.spaces.length);
 }
 
+export function getStrategy(strategies, spaces, id) {
```

Comment:
Would be better if we dont need to input all strategies just to return one
```suggestion
export function getStrategy(strategy, spaces) {
```

Reply:
We could reuse this function in `filterStrategies` to avoid duplicate code